src/common/ITEMS.js

```diff
@@ -36,7 +36,7 @@ const ITEMS = {
   },
   ARCHIMONDES_HATRED_REBORN: {
     id: 144249,
-    absorbId: 235169,
+    //absorbId: 235169,
```

Comment:
Should be removed instead of commented out, otherwise this will remain here *forever*